### PR TITLE
R stable enable virtio video

### PR DESCRIPTION
**Pull request by Shao-Feng**

### CODE REVIEW

File: groups/codec2/configurable/init.rc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+on post-fs
+    chmod 0664 /dev/ion
+    chown system system /dev/ion
+    setprop debug.stagefright.ccodec 3
```
Comment (dyang23) [inline groups/codec2/configurable/init.rc:4]: debug.stagefright.ccodec should set to 4

**Comment by harvan** [comment]
we must append CONFIG_VIRTIO_VIDEO=y in groups/kernel/gmin64/config-lts/lts2019-chromium/cfc_diffconfig

**Comment by Shao-Feng** [comment]
Updated

**Comment by Shao-Feng** [comment]
Updated and verified in R_stable. Please help to review.

**Review by dyang23 — APPROVED** [review]

**Review by dyang23 — COMMENTED** [review]
please make sure don't touch camera property.
for cfc target debug.stagefright.ccodec set to 0, please make sure remove it , and set it to 4.

**Review by dyang23 — APPROVED** [review]
LGTM, thanks
@lchen43  @xihua-chen @YuanjunHuang
please review

**Review by YuanjunHuang — APPROVED** [review]
Thanks for your effort!

**Comment by Shao-Feng** [comment]
Hi @lchen43 @JeevakaPrabu @kaushlen @sgnanase Please help to review

**Review by lchen43 — APPROVED** [review]

**Pull request merged**